src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -653,5 +653,34 @@
   </main>
 </div>
 
+<!-- TEMPORARY -->
+<!-- =============== -->
+<script>
+  function checkCLPAuth() {
+    var clpPassphrase = prompt('This page is in BETA. Please enter the passphrase to access it:');
+
+    // Escape early if the passphrase is incorrect.
+    if (clpPassphrase !== 'clptest123') {
+      checkCLPAuth();
+      return;
+    }
```

Comment:
I think either a redirect to the homepage or using a while-loop instead of recursion would feel a little better for me for some reason. IDK if there could be a side effect of the recursion.

Reply:
redirect to homepage implemented